src/main.rs

```diff
@@ -73,6 +73,10 @@ impl CycleRule {
 
 struct Instruction<'a> {
     name: &'static str,
+    operation: &'static str,
+    syntax: &'static str,
+    attributes: &'static str,
+    description: &'static str,
     cycle_rules: &'a [CycleRule],
```

Comment:
I have changed this so Instruction now have an ``desc`` field instead. This will allow us to share the description between tables for 000/020/etc without duplicating it each time.